lib/Common/Common/NumberUtilities.inl

```diff
@@ -72,7 +72,7 @@ namespace Js
     NUMBER_UTIL_INLINE INT64 NumberUtilities::TryToInt64(double T1)
     {
         INT64 T4_64;
-#if defined(_M_IX86)
+#if defined(_M_IX86) && defined(_WIN32) // TODO: xplat support
```

Comment:
I'm confused- FLD and FISTTP are not understood by other x86 assemblers? Or is it that we need to pass in a switch to get them to understand x87 instructions? Because FLD is a very old instruction. FISTTP is relatively newer- I think -msse3 should support both. Maybe it should get wrapped in a different ifdef (something like _M_CHAKRA_SSE3 ifdef that the makefile can pass in along with -msse3 to the compiler?)

---
In reply to: [78863793](https://github.com/Microsoft/ChakraCore/pull/1540#discussion_r78863793) [](ancestors = 78863793)